addons/website_slides/models/slide_channel.py

```diff
@@ -52,17 +54,22 @@ def _recompute_completion(self):
              ('completed', '=', True),
              ('slide_id.is_published', '=', True),
              ('slide_id.active', '=', True)],
-            ['channel_id', 'partner_id'],
+            ['channel_id', 'partner_id', 'slide_id:array_agg'],
             groupby=['channel_id', 'partner_id'], lazy=False)
         mapped_data = dict()
         for item in read_group_res:
             mapped_data.setdefault(item['channel_id'][0], dict())
-            mapped_data[item['channel_id'][0]][item['partner_id'][0]] = item['__count']
+            # Get all the Completed slides By the Partner
+            mapped_data[item['channel_id'][0]][item['partner_id'][0]] = item['slide_id']
```

Comment:
`item['channel_id'][0]` is duplicated, better to create a variable and to use it
```
channel_id = item['channel_id'][0]
```